src/components/Content/ApplicationContentPanel/MetaSection.tsx

```diff
@@ -0,0 +1,44 @@
+import { Application } from 'common/typedefs';
+import useEntityContext from 'components/global/hooks/useEntityContext';
+
+import { FieldRow, Section } from '../common/grid';
+import { FieldNames } from '../types';
+
+const ApplicationFieldRow = (props: any) => (
+  <FieldRow fieldNameWidth={6} fieldContentWidth={10} {...props} />
+);
+
+const MetaSection = () => {
+  const { entity } = useEntityContext();
+  const application = entity?.item as Application;
```

Comment:
you could pull out all the fields here and give sensible defaults:
`const {id, name, ... } = entity?.item`

Reply:
stylistic choice for you. if application might have a lot of undefined values makes more sense to keep it as is and have the individual sections handle their `undefined` states

Reply:
I'll likely move towards this kinda thing when i get the EntityContext more fleshed out. Will keep in mind!